lib/autokey/configmanager/configmanager.py

```diff
@@ -722,6 +742,7 @@ def remove_all_temporary(self, folder=None, in_temp_parent=False):
                 pass
             self.remove_all_temporary(subfolder, in_temp_parent)
 
+    # TODO: merge this function with model.Phrase.unset_hotkey
```

Comment:
This is actually member of `AbstractHotkey`, where you added it.

I think I remember a somewhat similar situation with comments that are dead-on-arrival ;-)

Reply:
hahahaha, had to `blame` to check myself.